Microsoft.DotNet.Interactive/LanguageService/MarkupContent.cs

```diff
@@ -0,0 +1,11 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.DotNet.Interactive.LanguageService
+{
+    public class MarkupContent
+    {
+        public MarkupKind Kind { get; set; }
```

Comment:
How about a mime type? :smile:

Reply:
I'm trying to stick to the LSP spec from [here](https://microsoft.github.io/language-server-protocol/specification#textDocument_hover).